apps/api/src/app/notification-template/usecases/update-notification-template/update-notification-template.usecase.ts

```diff
@@ -193,6 +193,7 @@ export class UpdateNotificationTemplate {
               title: message.template.title,
               preheader: message.template.preheader,
               senderName: message.template.senderName,
+              actor: message.template.actor,
```

Comment:
I didn't really feel it warrants a test, as it was just not added here when the actor field was added. The real danger/bug to test here is to make sure new fields are added to all message creation or update. Adding a test specifically for actor will not have any real benefit IMO.

Reply:
Without your fix, the users won't be able to add the new `in-app` steps with the `avatar` in the existing workflow. Doesn't this warrant a test? 
I'm not saying just to test the actor, but to test adding the `in-app` step and/or any other when the template exists, if we are missing this test. It will be super helpful and will cover this and other functionality.